src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -463,3 +463,103 @@
 		color: #405bfb;
 	}
 }
+
+.tec-tickets__admin-settings-tickets-commerce-section-menu {
+	padding-top: var(--tec-spacer-6);
+
+	a {
+		border-bottom-color: #c4c4c4;
+		border-bottom-style: solid;
+		border-bottom-width: 1px;
+		color: #23282d;
+		display: inline-block;
+		font-size: var(--tec-font-size-2);
+		margin-right: var(--tec-spacer-5);
+		padding-bottom: var(--tec-spacer-1);
+		text-decoration: none;
+	}
+
+	a.active {
+		font-weight: bold;
+	}
+
+	a.active,
+	a:hover {
+		border-bottom-color: #50b078;
+		border-bottom-width: var(--tec-spacer-0);
+	}
+}
+
+.tec-tickets__admin-settings-featured {
+	background-color: var(--tec-color-background);
+	border: solid 1px var(--tec-color-border-default);
+	border-radius: 8px;
+	padding: var(--tec-spacer-8);
+}
+
+.tec-tickets__admin-settings-featured-title {
+	background-color: transparent;
+	font-size: var(--tec-font-size-5);
+	font-weight: bold;
+	margin-bottom: var(--tec-spacer-2);
+	padding: var(--tec-spacer-1) 0;
+}
+
+.tec-tickets__admin-settings-featured-description {
+	margin-bottom: var(--tec-spacer-4);
+}
+
+.tec-tickets__admin-settings-featured-link {
+	align-items: center;
+	display: flex;
+}
+
+.tec-tickets__admin-settings-featured-link-url {
+	color: #405bfb;
+	padding: 0 var(--tec-spacer-1);
+}
+
+.tec-tickets__admin-settings-tickets-commerce-gateways-item {
+	align-items: center;
+	display: flex;
+	margin-bottom: var(--tec-spacer-7);
+	padding-bottom: var(--tec-spacer-0);
+}
+
+.tec-tickets__admin-settings-tickets-commerce-gateways-item-toggle {
+	margin-right: 35px;
+}
+
+.tec-tickets__admin-settings-tickets-commerce-gateways-item-brand {
+	flex-grow: 1;
+}
+
+.tec-tickets__admin-settings-tickets-commerce-gateways-item-brand-logo {
+	img {
```

Comment:
The same comment as the `a` above. Normally the way to go would be having `.tec-tickets__admin-settings-tickets-commerce-gateways-item-brand-logo-image` and use that one over the `img` element.